mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
+Scan Width,Width of the scan for GeoMx Analysis,,,FALSE,,,,HTAN,
+Scan Offset X,Offset X of the scan for GeoMx Analysis,,,FALSE,,,,HTAN,
+Scan Offset Y,Offset Y of the scan for GeoMx Analysis,,,FALSE,,,,HTAN,
+Binding Density,The binding density as reported by the application,,,FALSE,,,,HTAN,
+Positive norm factor,The Positive Control Normalization factor calculated using pos-hyb controls,,,FALSE,,,,HTAN,
+Surface area,Surface area of the ROI in square microns (µm2),,,FALSE,,,,HTAN,
+Nuclei count,Number of nuclei detected in the segment (if applicable),,,FALSE,,,,HTAN,
+Tissue Stain,e.g. CD45 or PanCK (if masking was performed),,,FALSE,,,,HTAN,
+Slide name,"Similar to a Run ID, the slide name indicates the slide a given ROI is linked to (as reported in Segment Summary).",,,FALSE,,,,HTAN,
+Raw reads,"Reads not yet analyzed in any way to be used for data analysis. The number of reads that pass filter from the flow cell represented in the FASTQ file. ",,,FALSE,,,,HTAN,
+Stitched reads,"Represents consensus from the overlapping sequence of read 1 and 2. This is a % of the aligned reads that were overlapped and consensus confirmed, usually upward of 80% but less in terms of number of reads than aligned reads",,,FALSE,,,,HTAN,
+Aligned reads,Is a sequence that has been aligned to a gene/probe. Typically these reads can number from the hundreds of thousands to tens of millions. In GeoMx alignment is via mapping the RTS ID to a white list of sequences that represent targets.,,,FALSE,,,,HTAN,
+Deduplicated reads,Is the replacement of blocks of duplicate data with a Virtual Index Pointer linking the new sub-block to the existing block of data in a duplicate repository. This is used to reduce the amount of space need to store the data.,,,FALSE,,,,HTAN,
+In Situ Negative median,Is the median of all negative control probes for a given segment. A measure of signal to background for each segment.,,,FALSE,,,,HTAN,
+Biological probe median,Is the median count from all probes except the negative control probes. A measure of signal to background for each segment,,,FALSE,,,,HTAN,
+Sequencing Saturation,"The fraction of reads originating from an already-observed UMI. This is a function of library complexity and sequencing depth. More specifically, this is the fraction of confidently mapped, valid spot-barcode, valid UMI reads that had a non-unique (spot-barcode, UMI, gene).",,,FALSE,,,,HTAN,
+MapQ30,Number of reads with Quality >= 30.,,,FALSE,,,,HTAN,
+Imaging Channel Metadata,Channel-level Metadata Attributes,,"Component, Channel Metadata ID, Channel ID, Channel Name, Channel Passed QC, Cycle Number, Sub Cycle Number, Antibody Role, Target Name, Antibody Name, RRID identifier, Fluorophore, Clone, Lot, Vendor, Catalog Number, Excitation Wavelength, Emission Wavelength, Excitation Bandwidth, Emission Bandwidth, Metal Isotope Element, Metal Isotope Mass, Oligo Barcode Upper Strand, Oligo Barcode Lower Strand, Dilution, Concentration",FALSE,,,,HTAN,
```

Comment:
I was going back and forth on including this, but as I mentioned above, I imagine that this information would be helpful for future reuse of the imaging data.

Reply:
Agree. But I wonder if these metadata come as a separate csv file like the `config.ini` file. In that case capturing the metadata file should suffice and we wont need these columns for each file.

Reply:
"Imaging Channel Metadata" is defined on line 154 as a separate metadata file, so the information can be collected and then referenced by other manifests.

Reply:
Thanks for the insight! We are definitely expecting IF, so I'll look into other approaches for capturing this info

Reply:
Propose replacing `Imaging Channel Metadata` with `Imaging Channel Names` which is a list like entry of the channel names / targets used. For stretch could validate these with valid values.